File: Examples/VisualBasic/Programming-Documents/Fields/InsertFormFields.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Words
Imports Aspose.Words.Fields
Public Class InsertFormFields
    Public Shared Sub Run()
        ' ExStart:InsertFormFields
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithFields()

        Dim doc As New Document()
        Dim builder As New DocumentBuilder(doc)

        Dim items As String() = {"One", "Two", "Three"}
        builder.InsertComboBox("DropDown", items, 0)
        ' ExEnd:InsertFormFields
        Console.WriteLine(vbLf & "Form fields inserted successfully.")
    End Sub
End Class

File: Examples/CSharp/Programming-Documents/Fields/InsertFormFields.cs
using System;
using System.Collections;
using System.IO;
using Aspose.Words;
using Aspose.Words.Fields;
namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_with_Fields
{
    class InsertFormFields
    {
        public static void Run()
        {
            //ExStart:InsertFormFields
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithFields();

            Document doc = new Document();
            DocumentBuilder builder = new DocumentBuilder(doc);

            string[] items = { "One", "Two", "Three" };
            builder.InsertComboBox("DropDown", items, 0);            
            //ExEnd:InsertFormFields
            Console.WriteLine("\nForm fields inserted successfully.");
        }
    }
}
